src/components/navBar.jsx

```diff
@@ -18,38 +19,40 @@ const NavBar = ({ user }) => {
       >
         <span className="navbar-toggler-icon" />
       </button>
-      <div className="collapse navbar-collapse" id="navbarNavAltMarkup">
-        <div className="navbar-nav">
-          {/* <NavLink className="nav-item nav-link" to="/dog">
-            Profil psa
-          </NavLink> */}
-          {!user && (
-            <React.Fragment>
-              <NavLink className="nav-item nav-link" to="/shelter">
-                Registrácia útulku
-              </NavLink>
-              <NavLink className="nav-item nav-link" to="/login">
-                Prihlásanie
-              </NavLink>
-              <NavLink className="nav-item nav-link" to="/register">
-                Registrácia používateľa
-              </NavLink>
-            </React.Fragment>
-          )}
-          {user && (
-            <React.Fragment>
-              <NavLink className="nav-item nav-link" to="/profile">
-                {user.name}
-              </NavLink>
-              <NavLink className="nav-item nav-link" to="/logout">
-                Odhlásiť
-              </NavLink>
-            </React.Fragment>
-          )}
-        </div>
-      </div>
+      <AuthorizationContext.Consumer>
+      {({loggedIn, userFirstName}) => (
+              <div className="collapse navbar-collapse" id="navbarNavAltMarkup">
+              <div className="navbar-nav">
+                {loggedIn !== true && (
```

Comment:
len maly tip, da sa to zapisat kratsie

```suggestion
                {!loggedIn && (
```

Reply:
navbar bude prerobeny ale dakujem za tip :)